### DISCUSSION TITLE
how can I change the design?

### DISCUSSION THREAD

**Discussion by phoenixqss** [AzuraCast/AzuraCast#4376] [General]
Help please! How can I change the default image on the Azuracast home page? in the Custom Branding section, the design changes only "Custom CSS for internal pages" ((((( "Custom CSS for public pages" - does not apply my design(((( how to get access to the catalog with pictures and replace them with your own using ftp? The path /var/azuracast/??????

**Comment by ivellios1988** [comment]
I use custom CSS for public pages, here's my code:
`div.public-page { background: #0c1320; }`
and it also works on my login page. The background is changed and the mascot is no longer there.
